plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/util/CodeWhispererConstants.kt

```diff
@@ -50,6 +50,7 @@ object CodeWhispererConstants {
     const val CODEWHISPERER_SUPPORTED_LANG_URI = "https://docs.aws.amazon.com/codewhisperer/latest/userguide/language-ide-support.html"
     const val CODEWHISPERER_CODE_SCAN_LEARN_MORE_URI = "https://docs.aws.amazon.com/codewhisperer/latest/userguide/security-scans.html"
     const val CODEWHISPERER_ONBOARDING_DOCUMENTATION_URI = "https://docs.aws.amazon.com/codewhisperer/latest/userguide/features.html"
+    const val CODEWHISPERER_GITHUB_URI = "https://github.com/aws/aws-toolkit-jetbrains"
```

Comment:
Should this be https://github.com/aws/aws-toolkit-jetbrains/issues?

Reply:
maybe pull ideas from https://github.com/aws/aws-toolkit-jetbrains/pull/3066

Reply:
I will change it to this if this makes more sense.